config/features.yml

```
@@ -297,3 +297,13 @@ features:
     enable_in_development: true
     description: >
       Ask for a password when an encrypted PDF is detected before uploading
+  education_reports_cleanup:
+    actor_type: user
+    enable_in_development: true
+    description: >
+      Updates to the daily education reports to remove old data that isn't needed in the new fiscal year
```

```suggestion
      Disables rendering of old data from the daily education reports that isn't needed in the new fiscal year
```

Minor nitpick here to more precisely define what the toggle *does* - feel free to wordsmith a bit more